qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -4,6 +4,7 @@
 
 POOL=rbd
 IMAGE=testrbdnbd$$
+TOOLARGE_IMAGE=${IMAGE}_large
```

Comment:
Minor: suggest adding an underscore between "TOO" and "LARGE" to make it clearer